content/en/v3/admin/platforms/k3s/_index.md

```diff
@@ -34,26 +34,47 @@ If you dont have an existing k3s cluster, you can install one by running:
 curl -sfL https://get.k3s.io | INSTALL_K3S_CHANNEL=v1.21 sh -s - --write-kubeconfig-mode 644
 sudo cp /etc/rancher/k3s/k3s.yaml ~/.kube/k3s-config
 # Set it also in the bashrc or zshrc file, or you can flatten both of these configs into a single file
-KUBECONFIG=~/.kube/config:~/.kube/k3s-config
+export KUBECONFIG=~/.kube/config:~/.kube/k3s-config
+
 ```
 
 To verify that k3s has been installed successfully, and configured run:
 
 ```bash
 kubectl get nodes
+```
+
+ - You will need to open multiple terminals later, so setting these env variables in the bashrc or zshrc might help you
+
+```bash
+nano ~/.bashrc
+#go to the end of the file and paste the export command
+export KUBECONFIG=~/.kube/config:~/.kube/k3s-config
+```
+
+**Optional**
+
+- If the above method didn't work, copy the configurations to the ~/.kube/config instead (if you don't have any other clusters, that should be fine)
+```bash
+sudo rm ~/.kube/k3s-config  #to make k3s uses ~/.kube/config
+sudo cp /etc/rancher/k3s/k3s.yaml ~/.kube/config
+```
+- If still got permession denied
+```bash
+sudo chmod 777 ~/.kube/config #warning: this maybe vulnerable to multiple users 
 ```
 
 This value of the node will be used later during installation and configuring of Jenkins X.
 
-Check [k3s install guide](https://rancher.com/docs/k3s/latest/en/installation/) for more installation options.
+Check [k3s install guide](https://rancher.com/docs/k3s/latest/en/installation/) for more installation options but keep in mind we don't support Kubernetes 1.22+ yet
 
 #### Vault
 
 Install vault cli.
 Refer to the [vault docs](https://www.vaultproject.io/docs/install) on how to install vault for your platform.
 
 Make sure you have vault running in a docker container with kubernetes auth enabled.
-
+You need to install [docker](https://docs.docker.com/engine/install/) and [manage it as a non root user](https://docs.docker.com/engine/install/linux-postinstall/)
```

Comment:
Let's add this as a separate pre-req.
So now there are 3 pre-reqs:
- k3s
- docker
- vault